templates/odoo/index.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  Output,
+  randomPassword,
+  randomString,
+  Services,
+} from "~templates-utils";
+import { Input } from "./meta";
+
+export function generate(input: Input): Output {
+  const services: Services = [];
+  const databasePassword = randomPassword();
+  const hashSalt = randomString();
+
+  services.push({
+    type: "app",
+    data: {
+      projectName: input.projectName,
+      serviceName: input.appServiceName,
+      source: {
+        type: "image",
+        image: input.appServiceImage,
+      },
+      proxy: { port: 8069, secure: true },
+      env: [
+        `HOST=${input.projectName}_${input.databaseServiceName}`,
+        `USER=odoo`,
+        `PASSWORD=${databasePassword}`,
+        `PORT=5432`,
+      ].join("\n"),
+      mounts: [
+        {
+          type: "volume",
+          name: "odoo-web-data",
+          mountPath: "/var/lib/odoo",
+        },
+        {
+          type: "volume",
+          name: "odoo-config",
+          mountPath: "/etc/odoo",
+        },
+        {
+          type: "volume",
+          name: "odoo-addons",
+          mountPath: "/mnt/extra-addons",
+        },
+      ],
+    },
+  });
+  services.push({
+    type: "app",
```

Comment:
Any reason for not using the dedicated service for postgres?

Reply:
The reason is becuase Odoo would not work with the postgres username, and I saw no way to change it with the dedicated service